lib/render.js

```diff
@@ -47,21 +47,14 @@ module.exports = function(options, emitter) {
   var stdin = options.stdin;
 
   var success = function(result) {
-    var todo = 1;
```

Comment:
The `done` still needs to be fired. It's used in other places. Please undo these changes and just remove 

```
emitter.on('done', function() {		
    // noop		
});
```